Gruntfile.js

```diff
@@ -3,7 +3,7 @@ module.exports = function(grunt) {
   // Project configuration.
   grunt.initConfig({
     jshint: {
-      files: ['Gruntfile.js', 'lib/**/*.js', 'test/**/*.js'],
+      files: ['Gruntfile.js', 'bin/**/*', 'lib/**/*.js', 'test/**/*.js'],
```

Comment:
Oh yikes. This is on my older Grunt based linting style. I'll submit a minor upgrade to `master` to resolve this x_x